README.md

```diff
@@ -1,38 +1,51 @@
 # Rosetta Kinase CM
+This repository is for our project <b><i>Comparative Modeling Pipeline of Protein Kinase Inhibitors</i></b>. 
 
-<!-- <figure class="image">
+
+<figure class="image">
   <img src="pipeline.png">
-  <figcaption>Figure: Overview of Rosetta Kinase CM pipeline</figcaption>
-</figure> -->
+  <figcaption>Figure: Overview of KinBADL pipeline</figcaption>
+</figure>
 
-## Dependency
-* Tested with [Python 3.7](https://www.python.org/downloads/) (Pandas) 
+## Dependencies
+* [Python 3.7](https://www.python.org/downloads/) (Pandas) 
 * [Rosetta](https://www.rosettacommons.org/software/license-and-download) Software Suite 
 * [PyRosetta](http://www.pyrosetta.org/) Software Suite
 * [OpenEye](https://www.eyesopen.com/) Software Suite
 * [EMBOSS](http://emboss.open-bio.org/html/use/ch02s07.html) Software Suite
 
-Initial folder should contain these three files and similar naming convention,
-
 ```
 2W1C_A_L0C
 ├── 2W1C_A.fasta
 ├── 2W1C_A_L0C.pdb
 └── L0C.smi
 ```
-Here are the STEPS and SCRIPTS used for the comparative modeling pipeline approach,
-
-## 1. Conformer generation and ligand alignment (modeling_script.py)
-* First, the maximum number of conformers will be generated for a given SMILES using OMEGA. The output will be a single SDF file with the maximum number of conformers.
-* Second, the single SDF file will be used for the alignment of a query (Molecule of interest) and database (in-house active kinase ligand template library) molecules using ROCS. The output will be each template aligned query molecule.
-* Third, combining the report files of each template aligned query molecule into a single report file.
-* Fourth, based on the single report file, the top 100 conformers for the given SMILES are selected.
-* Fifth, SDF to PARAMS will be generated using 100 conformers for Rosetta Minimization step. 
+
+
+The initial folder should contain three files and following the naming convention
+- ```2W1C_A_L0C.fasta``` is the FASTA protein file for the kinase 2W1C
+- ```2W1C_A_L0C.pdb``` is the PDB structure of 2W1C in complex with the L0C ligand
```

Comment:
@wgabrielong Shall we add one more note to specify that it can be an empty file as well?